website/src/redux/reducers/ElementReducer.jsx

```diff
@@ -0,0 +1,86 @@
+/**
+ * @file Reducers having to do with the position of elements.
+ */
+
+import {
+  INIT_ELEMENTS, UPDATE_POSITION, UPDATE_SIZE, UPDATE_ROTATION, TARGET_ELEMENT,
+} from './../actions/ActionConstants';
+
+/**
+ * Inserts payload information into state, making a new object.
+ * @param {Object} state The state of the store.
+ * @param {Object} elementId ID of the element affected.
+ * @param {Object} payload The updated information to insert.
+ * @param {String} objectField The field of the element to update. If left as
+ *                             null this indicates the payload already contains
+ *                             the field.
+ * @returns {Object} The new state object.
+ */
+function copyPayloadInfo(state, elementId, payload, objectField = null) {
+  if(!payload) {
+    return state;
+  }
+  // Copy the appropriate field
+  let fieldToUpdate = {};
+  if (objectField === null) {
+    fieldToUpdate = payload;
+  } else {
+    fieldToUpdate[objectField] = Object.assign({}, payload);
+  }
+  // Make copy of the element with new field.
+  const elemToUpdate = {};
+  elemToUpdate[elementId] = Object.assign({}, state.elements[elementId],
+    fieldToUpdate);
+  // Make new state with new copy of element inserted
+  const updatedElements = Object.assign({}, state.elements, elemToUpdate);
+  return Object.assign({}, state, {
+    elements: updatedElements,
+  });
+}
+
+/**
+ * Update the position in firebase.
+ * @param {Object} state The state of the store.
+ * @param {Object} action action to be performed.
+ * @returns {Object} The new state object.
+ */
+export const updateElementReducer = (state = { elements: {} }, action) => {
+  switch (action.type) {
+    case INIT_ELEMENTS:
+      return Object.assign({}, state, {
+        elements: action.elements,
+      }, {
+        targeted: null,
+      });
+
+    case UPDATE_POSITION:
+      return copyPayloadInfo(state, action.elementId, action.payload,
+        'position');
```

Comment:
True, I made them constants in this file for now. Later when state is refactored we should make a new file with constants of the different state fields.